src/controllers/automation/automationGridCtrl.js

```diff
@@ -21,6 +21,7 @@
     $scope.rows = 10;
     $scope.tasksQuantity = '';
     $scope.totalTasks = 0;
+    $scope.createNewAutomationView =  $window.location.href.indexOf("/selectAutomation") > 0;
```

Comment:
Maybe this flag could be named: `isCreateNewAutomationViewVisible` or something like that... the current name confused me, what do you think? With the first look, I believed it would be a path to a view 😄